src/components/TransferRow.svelte

```diff
@@ -0,0 +1,65 @@
+<script>
+    import { sanitizeAddress } from '$lib/utils';
+    import dayjs from 'dayjs';
+    import relativeTime from 'dayjs/plugin/relativeTime';
+    import updateLocale from 'dayjs/plugin/updateLocale';
+    import OpenIcon from '$lib/assets/open-in-newtab.svg';
+
+    dayjs.extend(relativeTime);
+    dayjs.extend(updateLocale);
+    dayjs.updateLocale('en', {
+        relativeTime: {
+            future: 'in %s',
+            past: '%s ago',
+            s: 'a few seconds',
+            m: 'a minute',
+            mm: '%d min',
+            h: 'an hour',
+            hh: '%d hours',
+            d: 'a day',
+            dd: '%d days',
+            M: 'a month',
+            MM: '%d months',
+            y: 'a year',
+            yy: '%d years',
+        },
+    });
+
+    export let receiver;
+    export let timestamp;
+
+    let url = `https://explorer.testnet.evm.eosnetwork.com/address/${receiver}`;
```

Comment:
URL inside is ok for now. For stability reasons we may setup our own testnet, and need pull this URL from env or constants file.